rust/azure_iot_operations_services/src/leased_lock.rs

```diff
@@ -42,13 +42,13 @@ impl From<state_store::Error> for Error {
     }
 }
 
-/// Represents the kinds of errors that occur in the Azure IoT Operations Leased Lock implementation.
+/// Represents the kinds of errors that occur in the Azure IoT Operations Key Lease implementation.
 #[derive(Error, Debug)]
 #[allow(clippy::large_enum_variant)]
 pub enum ErrorKind {
     /// The lock is already in use by another holder.
     #[error("lock is already held by another holder")]
```

Comment:
I think this description will need to be updated

Reply:
Done.